File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.cs
using System;

public class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        string even = "";
        string odd = "";
        for (int i=0; i<num_list.Length; i++) {
            if (num_list[i]%2 == 0) {
                even += num_list[i].ToString();
            }
            else if (num_list[i]%2 == 1) {
                odd += num_list[i].ToString();
            }
        }
        return Int32.Parse(even) + Int32.Parse(odd) ;
    }
}

File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> num_list) 
{
    string odd_temp = "";
    string even_temp = "";
    for(int i=0; i<num_list.size(); i++)
    {
        if(num_list[i]%2 == 1)
        {
            odd_temp += to_string(num_list[i]);
        }
        else
        {
            even_temp += to_string(num_list[i]);
        }
    }
    return stoi(odd_temp) + stoi(even_temp);
}